content/release-notes/2023-11-21-console.md

```diff
@@ -0,0 +1,10 @@
+---
+description: Read about the latest fixes and improvements in Neon Serverless Postgres.
+---
+
+### Fixes & improvements
+
+- Control Plane: Raised the PgBouncer `default_pool_size` limit, which defines the number of server connections allowed per user/database pair, from 16 to 64 to avoid running out of available server connections.
```

Comment:
```suggestion
- Control Plane: Raised the PgBouncer `default_pool_size` limit from 16 to 64 to prevent running out of available server connections. This limit defines the number of server connections allowed per user/database pair.
```